retrybp/filters.go

```diff
@@ -0,0 +1,134 @@
+package retrybp
+
+import (
+	"context"
+	"errors"
+	"net"
+
+	retry "github.com/avast/retry-go"
+
+	"github.com/reddit/baseplate.go/clientpool"
+)
+
+var (
+	// DefaultFilterDecision is the default decision returned by Filters at the end
+	// of the FilterChain.
+	DefaultFilterDecision = false
+)
```

Comment:
nit: Now this is a primitive type, it can be `const` instead of `var` now.

Reply:
The point of making it `var` was so you _could_ change it if you wanted to.

Reply:
That can be achieved by just add another one to the chain, but fine :)

Reply:
although, you can just terminate the filters with one that always returns true if you really want that... I'll change this.